connector/exceptionsconnector/README.md

```diff
@@ -0,0 +1,74 @@
+# Exceptions Connector
+
+| Status                   |               |
+| ------------------------ |---------------|
+| Stability                | [development] |
+| Supported pipeline types | See [Supported Pipeline Types](#supported-pipeline-types)  |
```

Comment:
I think you should just enter [metrics] here. Please also look at how to generate this from metadata.yaml. See https://github.com/open-telemetry/opentelemetry-collector-contrib/issues/21213 for reference.

Reply:
I added the metadata.yaml file and I runed `make generate` but nothing change. I'm missing anything?

Reply:
I just did it, run `make generate` but not luck :shrug:

Reply:
You need to add this somewhere in your package:

```
//go:generate mdatagen metadata.yaml
```

Ideally, place this in the `factory.go`.